alchemist/src/orchid/resources/wiki/usage/maps.md

```diff
@@ -0,0 +1,81 @@
+---
+
+title: Maps and GPS traces
+
+---
+
```

Comment:
Introduction: Alchemist can simulate real-world scenarios using map data from openstreetmaps... etc.

Reply:
I think it was already done